.github/scripts/print_latest_commits.py

```diff
@@ -83,16 +85,26 @@ def isGreen(commit: str, results: Dict[str, Any]) -> Any:
             return workflowName + " checks were not successful"
     return True
 
+def get_latest_green_commit(commits: List[str], results: Dict[str, Any]) -> Any:
+    for commit in commits:
+        if isGreen(commit, results) is True:
```

Comment:
```suggestion
        if isGreen(commit, results):
```
classic

Reply:
Oh wait I see, cuz you return a string otherwise. 

I dislike the confusion where it returns a string OR True--maybe a better way is for isGreen to return a Tuple[bool, str]? So we can just have

```suggestion
        if isGreen(commit, results)[0]:
```
here?

Reply:
yep i changed the return type to a tuple! so it will still have an error message but checking isgreen is more straightforward